llvm/lib/Target/AMDGPU/SIRegisterInfo.td

```diff
@@ -881,10 +885,12 @@ def SReg_1 : SIRegisterClass<"AMDGPU", [i1], 32,
   let HasSGPR = 1;
 }
 
+
 multiclass SRegClass<int numRegs,
                      list<ValueType> regTypes,
                      SIRegisterTuples regList,
                      SIRegisterTuples ttmpList = regList,
+                     bit hasNull = 0,
                      int copyCost = !sra(!add(numRegs, 1), 1)> {
```

Comment:
Not really related to your patch, but copyCost could be a defvar if no-one is setting it.

Reply:
Agreed, but I think I'll leave it as is unless you think it should be fixed.

Reply:
It's fine to leave it as-is in this patch.